tests/etl/test_perf_data_load.py

```diff
@@ -0,0 +1,221 @@
+    return {
+        'job_guid': 'fake_job_guid',
+        'name': 'test',
+        'type': 'test',
+        'blob': {
+            'framework': {'name': FRAMEWORK_NAME},
+            'suites': [
+                {
+                    'name': 'cheezburger metrics',
+                    'extraOptions': ['shell', 'e10s'],
+                    'lowerIsBetter': True,
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT,
+                            'lowerIsBetter': True
+                        },
+                        {
+                            'name': 'test2',
+                            'value': 30.0,
+                            'unit': MEASUREMENT_UNIT,
+                            'lowerIsBetter': False
+                        },
+                        {
+                            'name': 'test3',
+                            'value': 40.0,
+                            'unit': MEASUREMENT_UNIT,
+                        }
+                    ]
+                },
+                {
+                    'name': 'cheezburger metrics 2',
+                    'lowerIsBetter': False,
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT,
+                        }
+                    ]
+                },
+                {
+                    'name': 'cheezburger metrics 3',
+                    'value': 10.0,
+                    'unit': MEASUREMENT_UNIT,
+                    'subtests': [
+                        {
+                            'name': 'test1',
+                            'value': 20.0,
+                            'unit': MEASUREMENT_UNIT
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+
+
+@pytest.fixture
+def datum_with_new_unit():
+    return {
+        'job_guid': 'fake_job_guid',
+        'name': 'test',
+        'type': 'test',
+        'blob': {
+            'framework': {'name': FRAMEWORK_NAME},
+            'suites': [
+                {
+                    'name': 'cheezburger metrics',
+                    'extraOptions': ['shell', 'e10s'],
+                    'lowerIsBetter': True,
+                    'value': 10.0,
+                    'unit': UPDATED_MEASUREMENT_UNIT,
+                    'subtests': [
+                            {
+                                'name': 'test1',
+                                'value': 20.0,
+                                'unit': UPDATED_MEASUREMENT_UNIT,
+                                'lowerIsBetter': True
+                            },
```

Comment:
It would be better to add a clone of the other 2 subtests here.

Reply:
Ok, will do.